specs/protocol/isthmus/configurability.md

```diff
@@ -0,0 +1,24 @@
+# Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [`SystemConfig`](#systemconfig)
+  - [`ConfigUpdate`](#configupdate)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## `SystemConfig`
+
+### `ConfigUpdate`
+
+The following `ConfigUpdate` enum is defined where the `CONFIG_VERSION` is `uint256(1)`:
+
+| Name | Value | Definition | Usage |
+| ---- | ----- | --- | -- |
+| `BATCHER` | `uint8(0)` | `abi.encode(address)` | Modifies the account that is authorized to progress the safe chain |
+| `FEE_SCALARS` | `uint8(1)` | `(uint256(0x01) << 248) \| (uint256(_configurablefeescalar) << 128 \| (uint256(_configurablefeeconstant) << 64 \| (uint256(_blobbasefeeScalar) << 32) \| _basefeeScalar` | Modifies the fee scalars |
```

Comment:
To keep it consistent with the (blob) base fee scalar variable names, the words `Scalar` and `Constant` should start with capital letters.
```suggestion
| `FEE_SCALARS` | `uint8(1)` | `(uint256(0x01) << 248) \| (uint256(_configurablefeeScalar) << 128 \| (uint256(_configurablefeeConstant) << 64 \| (uint256(_blobbasefeeScalar) << 32) \| _basefeeScalar` | Modifies the fee scalars |
```